aten/src/ATen/native/cpu/ActivationKernel.cpp

```diff
@@ -0,0 +1,484 @@
+#include "ATen/native/cpu/ActivationKernel.h"
+
+#include <algorithm>
+#include <iterator>
+#include <limits>
+#include <numeric>
+
+#include "ATen/Dispatch.h"
+#include "ATen/Parallel.h"
+#include "ATen/cpu/vec256/vec256.h"
+#include "ATen/optional.h"
+
+// NOTE: In general we avoid calls into cmath for code compiled with AVX/AVX2
+// This is because of SSE-AVX transitions and a bug in Glibc2.23
+// See https://bugs.launchpad.net/ubuntu/+source/glibc/+bug/1663280
+
+namespace at { namespace native {
+namespace {
+
+static tbb::affinity_partitioner ap;
+
+template <int64_t size>
+inline int64_t _leftover(int64_t x, int64_t y) {
+  if (x + size > y)
+    return y - x;
+  return size;
+}
```

Comment:
I can't see why do we need this function. Isn't this just `std::min(y - x, size)`?

Reply:
Sure, but this is important enough to not copy paste all over. Also I want to avoid calls into std:: to not encourage calls into cmath.

Reply:
I'm pretty sure that `std::min` on two integers doesn't use SSE 😄